sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -1478,15 +1478,15 @@ public partial class PairedRegion
     public partial class ParameterDefinitionsValue
     {
         public ParameterDefinitionsValue() { }
-        public System.Collections.Generic.IList<object> AllowedValues { get { throw null; } }
-        public object DefaultValue { get { throw null; } set { } }
+        public System.Collections.Generic.IList<System.BinaryData> AllowedValues { get { throw null; } }
+        public System.BinaryData DefaultValue { get { throw null; } set { } }
```

Comment:
These properties look like `any` instead of `anyObject` - therefore `any` is more suitable for this property.
This property is the default value of a property we defined in a policy definition - which is almost in all cases, is a primitive type
Please consider to add a configuration to make sure this property is an `any` instead of an `anyObject`